Documentation/Usage.md

```diff
@@ -143,84 +143,86 @@ To `import Clibgit`, the package manager requires that the libgit2 library has
 been installed by a system packager (eg. `apt`, `brew`, `yum`, etc.). The
 following files from the libgit2 system-package are of interest:
 
-    /usr/local/lib/libgit2.dylib      # .so on Linux
-    /usr/local/include/git2.h
+    /opt/homebrew/lib/libgit2.dylib      # .so on Linux
+    /opt/homebrew/include/git2.h
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+**Note:** the system library may be located elsewhere on your system, such as
+`/usr/local/`, or `/usr/`. The paths above are valid for Homebrew on macOS.
 
-Note that the system library may be located elsewhere on your system, such as
-`/usr/` rather than `/usr/local/`.
-
-Create a directory called `Clibgit` next to the `example` directory and
-initialize it as a package that builds a system module:
-
-    example$ cd ..
-    $ mkdir Clibgit
-    $ cd Clibgit
-    Clibgit$ swift package init --type empty
-
-This creates a `Package.swift` file in the directory.
-Edit `Package.swift` and add `pkgConfig` parameter:
+Now let's define `Clibgit` dependency in your `Package.swift` in the example project:
 
 ```swift
-// swift-tools-version:5.3
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        // Targets are the basic building blocks of a package, defining a module or a test suite.
+        // Targets can depend on other targets in this package and products from dependencies.
+        .executableTarget(
```

Comment:
it may be easier to understand if we do this in two steps, first introduce the "Clibgit" target, then show how to use it in the executable target. in other words, I suggest we split this example into two snippets, and build the knowledge in layers

Reply:
It did feel a little bit overwhelming when I tried to reproduce the steps, so I agree. Rewritten that in two steps, and cleaned up the `pkg-config` mention as well.